service/project.go

```diff
@@ -343,54 +388,6 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	var hook *model.GithubHook
```

Comment:
moved this code up so we could validate webhooks are enabled on the server side when enabling PR testing and the commit queue.